uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/components/leave-request-popup-details-tab.component.spec.js

```diff
@@ -1653,20 +1650,40 @@ define([
           delete toilRequest.toil_expiry_date;
 
           AbsenceType.canExpire.and.returnValue($q.resolve(true));
-          compileComponent({
-            mode: 'edit',
-            leaveType: 'toil',
-            request: toilRequest
+        });
+
+        describe('when request date changes for manager', function () {
```

Comment:
These tests needs to be put into their individual files once PCHR-3281 is merged, I am trying to get it merged by today.